tensorrt_llm/_torch/modules/fused_moe.py

```diff
@@ -232,6 +232,45 @@ class FusedMoE(nn.Module):
         reduce_results (bool): Whether to reduce the results across devices.
         model_config (ModelConfig): Configuration object for the model.
         enable_alltoall (bool): whether to enable alltoall instead of allgather/reducescatter
+
+    MoE Op:
+        cutlass Backend
+            In min latency mode:
+            Quant:
+                fp8 block scales:
+                    FusedMoE Op: dynamic quant + gemm1 + swiglu + gemm2 (return tensor list).
+                fp8 qdq, nvfp4
+                    FusedMoE Op: gemm1 + swiglu + gemm2 (return tensor list).
+
+            In maxthroughput mode:
+            Quant:
+                fp8 block scales:
+                    FusedMoE Op: dynamic quant + scatter + gemm1 + swiglu + gemm2 + finalizeMoeRoute (return one tensor)
+                p8 dqd, nvfp4
+                    FusedMoE Op: scatter + gemm1 + swiglu + gemm2 + finalizeMoeRoute (return one tensor)
+
+        trtllm_gen backend:
+            Only support min latency mode now.
+            Quant: fp8 block scales quant(1 * 128) and nvfp4 quant
+                FusedMoE Op: routing(topK, etc.) + scatter + gemm1 + swiglu + gemm2 + finalize MoeRoute
+
+    FusedMoE module:
+        cutlass Backend
+            min latency mode:
+                routing(topK, etc.) + FusedMoE Op
+                equals to: dynamic quant + routing(topK, etc.) + gemm1 + swiglu + gemm2
+
+            maxthroughput mode:
+                routing(topK, etc.) [+ dynamic quant for fp8 qdq and nvfp4 ] [+ allgather when dp enabled and nvfp4 allgather enabled]  + FusedMoe Op [+ allReduce is self.reduce_results] + reducescatter
+                equals to: dynamic quant + routing(topK, etc.) [+ allgather when dp enabled and nvfp4 allgather disabled] + scatter + gemm1 + swiglu + gemm2 + finalizeMoeRoute [+ allReduce is self.reduce_results] + reducescatter
+
+        trtllm_gen backend:
+            min latency mode:
```

Comment:
You should clarify that the flag `min_latency_mode` has no effect when trtllm_gen is used.